test/acceptor_SUITE.erl

```diff
@@ -51,6 +52,27 @@ groups() ->
 		tcp_many_listen_sockets_no_reuseport,
 		tcp_error_eaddrinuse,
 		tcp_error_eacces
+	]}, {tcp_socket, [
+		tcp_active_echo,
+		tcp_active_n_echo,
+		tcp_echo,
+		tcp_graceful,
+		tcp_inherit_options,
+		tcp_max_connections,
+		tcp_max_connections_and_beyond,
+		tcp_max_connections_infinity,
+		tcp_remove_connections,
+		tcp_remove_connections_acceptor_wakeup,
+		tcp_set_max_connections,
+		tcp_set_max_connections_clean,
+		tcp_getopts_capability,
+		tcp_getstat_capability,
+		tcp_upgrade,
+		%% @TODO: Enable when https://bugs.erlang.org/browse/ERL-1287 is fixed
```

Comment:
This has been fixed so I guess the tests can be enabled again.

Reply:
~Yes~ No... If I run them on OTP 24, they fail. I think I read somewhere (can't remember where) that while it was fixed in 23.something, it re-occurred in 24. I'm investigating, I don't have the needed version of 23 in kerl right now, it's currently building.

Reply:
On top of that, with 24 `tcp_error_eacces` fails with an obscure error message, but this one was definitely working before. Investigating that one, too, but it looks like this will need another report and fixing in OTP.

Reply:
Let me give you a hand? =)

I ran them on 23.3 for you, and while `tcp_10_acceptors_10_listen_sockets` ultimately fails, it is for another reason, _starting_ the listener actually works there. With 24, I can confirm that it doesn't start.

Reply:
@juhlig thanks 🥰 I'll collect the bits and pieces later and open a ticket at OTP.

Well... so I guess we can't merge this just yet, or if, only with the 2 tests plus the eacces one disabled... @essen?

Reply:
Right they are related to that raw issue, OK, should have checked more thoroughly... So we can leave those disabled until an OTP release fixes the issue and then enable them for 24.x+.

I think we can merge with the 2 tests + eacces disabled for now. But we should try them against current OTP master before opening any ticket there. Want to do that?

Reply:
> Want to do that?

Can do, but not today :)

Reply:
Oh, master build was faster than expected.

Ok, so on a quick glance, it all seems to work (except for what Jan pointed out, a test failing for other reasons than the socket backend) when run on current master, even the eacces test.
So... I'll need to fix that test (needs some thinking, but I'll manage. Tomorrow. For real this time 😅), and then we can enable them when 25 is out, or 24.1 if it's already fixed in there.

Reply:
Ok, the 3 tests are disabled with a todo note to re-enable them when fixed (probably 24.1), and the one test that wasn't working has been fixed (this post listen callback thing came at just the right time for this :)), but it is among the disabled ones.